packages/autorest.python/autorest/codegen/models/primitive_types.py

```diff
@@ -68,6 +68,10 @@ def get_json_template_representation(
     def default_template_representation_declaration(self) -> str:
         return self.get_declaration(self.docstring_type())
 
+    @property
+    def type_description(self) -> str:
```

Comment:
seems like we don't need this here. I'll do a little commit to remove this section

Reply:
oh i misread this, i thought it was just returning self.type_description. My bad